apps/drec-api/package.json

```diff
@@ -62,8 +62,9 @@
     "bcryptjs": "2.4.3",
     "class-validator": "0.13.1",
     "class-transformer": "0.3.1",
+    "clean-deep": "^3.4.0",
```

Comment:
please lock version.

rush has also `--exact` method if you dont want rush to add "^" during `rush add`